Makefile

```diff
@@ -75,14 +75,7 @@ cobertura:
 		grep -v zz_generated.deepcopy | \
 		$(GOCOVER_COBERTURA) > $(GO_TEST_OUTPUT)/cobertura-coverage.xml
 
-crds.clean:
```

Comment:
why we remove crds.clean ?

Reply:
With the new version of controller-tools, it's not necessary anymore.